examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>These calibrations are vended as <code>PulseSequence</code> objects and contain a list of low-level instructions that are executed for that gate. Obtaining the accurate sequence that used by the provider offers many advantages. First, it is one of the best ways to understand how quantum computers work at the most fundamental level. It also allows you to improve the results of your circuits by deploying error mitigation techniques such as zero noise extrapolation, by stretching all the pulses of specific gates. </p><p> </p><p><code>Native gate calibrations are available with Rigetti's Aspen M-3</code> -&gt; Is it possible to retrieve the devices which provide native gate calibrations programmatically? Hardcoding device names here means we have to come back and update this notebooks every time a new device supports this, or is deprecated.</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>